script/brave_license_helper.py

```diff
@@ -31,10 +32,120 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
+        os.path.join('brave', 'third_party', 'rust', 'blake2b_simd'),
+        os.path.join('brave', 'third_party', 'rust', 'cid'),
+        os.path.join('brave', 'third_party', 'rust', 'convert_case'),
+        os.path.join('brave', 'third_party', 'rust', 'crunchy'),
+        os.path.join('brave', 'third_party', 'rust', 'derivation_path'),
+        os.path.join('brave', 'third_party', 'rust', 'ed25519_dalek_bip32'),
+        os.path.join('brave', 'third_party', 'rust', 'ff_derive'),
+        os.path.join('brave', 'third_party', 'rust', 'forest_bigint'),
+        os.path.join('brave', 'third_party', 'rust', 'fvm_ipld_blockstore'),
+        os.path.join('brave', 'third_party', 'rust', 'fvm_ipld_encoding'),
+        os.path.join('brave', 'third_party', 'rust', 'fvm_shared'),
+        os.path.join('brave', 'third_party', 'rust', 'fxhash'),
+        os.path.join('brave', 'third_party', 'rust', 'halo2_gadgets'),
+        os.path.join('brave', 'third_party', 'rust', 'halo2_proofs'),
+        os.path.join('brave', 'third_party', 'rust', 'libsecp256k1_core'),
+        os.path.join('brave', 'third_party', 'rust', 'libsecp256k1_gen_ecmult'),
+        os.path.join('brave', 'third_party', 'rust',
+                     'libsecp256k1_gen_genmult'),
+        os.path.join('brave', 'third_party', 'rust', 'mac'),
+        os.path.join('brave', 'third_party', 'rust', 'multibase'),
+        os.path.join('brave', 'third_party', 'rust', 'multihash'),
+        os.path.join('brave', 'third_party', 'rust', 'multihash_derive'),
+        os.path.join('brave', 'third_party', 'rust', 'phf'),
+        os.path.join('brave', 'third_party', 'rust', 'phf_codegen'),
+        os.path.join('brave', 'third_party', 'rust', 'phf_generator'),
+        os.path.join('brave', 'third_party', 'rust', 'phf_generator'),
+        os.path.join('brave', 'third_party', 'rust', 'phf_macros'),
+        os.path.join('brave', 'third_party', 'rust', 'phf_shared'),
+        os.path.join('brave', 'third_party', 'rust', 'phf_shared'),
+        os.path.join('brave', 'third_party', 'rust', 'seahash'),
+        os.path.join('brave', 'third_party', 'rust', 'selectors'),
+        os.path.join('brave', 'third_party', 'rust', 'serde_tuple'),
+        os.path.join('brave', 'third_party', 'rust', 'serde_tuple_macros'),
+        os.path.join('brave', 'third_party', 'rust', 'servo_arc'),
+        os.path.join('brave', 'third_party', 'rust', 'siphasher'),
+        os.path.join('brave', 'third_party', 'rust', 'thin_slice'),
+        os.path.join('brave', 'third_party', 'rust', 'uint'),
+        os.path.join('brave', 'third_party', 'rust', 'valuable'),
+        os.path.join('brave', 'third_party', 'rust', 'wasi'),
+        os.path.join('brave', 'third_party', 'rust',
+                     'winapi_i686_pc_windows_gnu'),
+        os.path.join('brave', 'third_party', 'rust',
+                     'winapi_x86_64_pc_windows_gnu'),
+
+        # Skip this Rust crate because it doesn't conform to normal layout (the
```

Comment:
Since it's just one crate, we could add a custom manual entry in `src/brave/third_party/` like we do for npm modules (e.g. https://github.com/brave/brave-core/tree/master/third_party/npm_jszip).

Reply:
Done in https://github.com/brave/brave-core/pull/23822/commits/4fe3aa4f19a3702d565bd3ab98f6717b9c2f81b3